### ISSUE TITLE
Include accessibility attributes

### ISSUE THREAD

**Issue by tmcw** [observablehq/inspector#189]
I received some useful feedback from someone who uses a screenreader about a notebook that I shared ([mastodon thread here](https://social.stealthy.club/@esoteric_programmer/statuses/01J8GR9J2DXXB2VN7W4GV5VTXQ)), which was interesting because, while it's pretty jumbled right now, screenreaders _can_ look into the iframe and read some of a page on Observable!

But on the other hand, the inspector is a stumbling block for screenreaders right now:

- The collapsed / expanded states are not accessible: it's a div with click events attached to it. Ideally this would include an [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) attribute, as well as `role=button` (it _could_ be a button element, but I think that would run into nesting rules  pretty quickly)
- The special `ƒ` is not accessible to screenreaders - they read the unicode description of it. I think maybe this would be something where it'd be good to have an aria-description, and possibly include the f as a css `::after` or something that isn't read as part of the page content.
